sdk/search/CONTRIBUTING.md

```diff
@@ -72,57 +72,54 @@ live tests against a pull request by commenting `/azp run net - search - tests`
 in the PR.
 
 ### Live Test Resources
-You'll need to pass the details for a subscription, service principal, resource
-group, and location via environment variables.  We set this up for live tests
-using the [test-resources.json](./test-resources.json) ARM template, but it's
-equivalent to the following steps if you want to run the live tests locally.
+To set up your Azure account to run live tests, you'll need to log into Azure,
```

Comment:
Should we add a blurb - perhaps even a link - to first install the Azure module? We didn't actually document to install the az CLI, but I'm open to adding something if this is to become our template for other services.

Reply:
Do we need these instructions in every service directory that uses live resource management? Can we centralize them and link to a central location?

Reply:
Yes I think we should be in the central location with a link and perhaps one example command line that is specialized for the service, but if it is only the service directory maybe that isn't very interesting.

Reply:
I mostly agree, but the output will be different for some services. I could make a comment to that affect - that people should copy the whole output and that the example is just that - example output that may vary (I'll probably just use this exactly as an example).

Actually, one other concern: our docs are a mess. Should this be an absolute URL to a central doc - perhaps a section in /CONTRIBUTING.md. Agreed?

Reply:
Discussed offline. I'll add a README.md to eng/common/TestResources (I'll sync up to azure-sdk-tools after we agree on content with this PR) and point to there from this README.md, and also link from the repo root's CONTRIBUTING.md.

/cc @danieljurek